mantidproject/mantid
# Adding multi plot to make elemental analysis v 2.0

A new elemental analysis has been added and the old one has been renamed `Elemental_Analysis_Old`. This new version is designed to be easier to maintain and develop. It also uses the new multiplotting widget. 

The code will be refactored and unit tests added as part of #24926

**Report to:** [user name]/[nobody]. 

**To test:**
Contact me and I will send you some data (not to be distributed) 

1. Open Elemental Analysis
2. Click manage user directories and add the directory with the data to the search paths
3. In the run box enter 2695 and click load - data should be loaded
4. Clicking combinations of detectors, checkboxes and the periodic table will add lines to the plot. You will need to have a detector clicked to make the plot appear. 


Fixes #24807 . 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
